### DISCUSSION TITLE
Is the privacy policy tenant(Dhawani) specific or instance(SunbirdSaas) specific

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#445] [General Discussion]
can we change the privacy policy or terms and conditions according to Tenanat?
Like Tenant, we have Prathama, Dhwani, Educate girls, etc.
If someone has logged in and the terms and conditions or privacy policy can be changed according to the tenant(if it is possible)
Like in the image below the name is **Diksha** so can we change it tenant specific like Dhwani or Prathama.
[@vinukumar-vs](https://github.com/vinukumar-vs)  [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)
![image](https://user-images.githubusercontent.com/109504832/228209037-22b07530-590f-4a85-932c-10379b956c25.png)

**Comment by nidhikachhawa24** [comment]
@rajeevsathish @Krishnaj20 @vinukumar-vs can anyone help on this?

**Reply by rajeevsathish** [reply]
@nidhikachhawa24 as of now this is only available from {{host}}/api/data/v1/system/settings/get/tncConfig this is not Tenanat specific

**Reply by Krishnaj20** [reply]
@rhwarrier @reshmi-nair - Can you confirm if each tenant can have their own T&C or it is at an instance level?

**Reply by Krishnaj20** [reply]
@rhwarrier @reshmi-nair - Gentle reminder: Can you check and respond please?

**Comment by rhwarrier** [comment]
This is currently at the instance level, is my understanding.
@reshmi-nair kindly confirm.

**Reply by reshmi-nair** [reply]
@Krishnaj20 @rajeevsathish @nidhikachhawa24 This is in instance level, not in tenant level.

**Reply by Krishnaj20** [reply]
thanks @reshmi-nair & @rhwarrier .

@rajeevsathish - pls note.
